scripts/tests-after-upgrade/tests-after-upgrade.sh

```diff
@@ -0,0 +1,286 @@
+#!/bin/bash
+# Copyright 2025 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ENVIRONMENT=${1:-local}
+ENV_FILE="scripts/tests-after-upgrade/${ENVIRONMENT}.env"
+
+set -euo pipefail
+source integration_tests/cli/common.sh
+  source "$ENV_FILE"
+
+if [ "$ENVIRONMENT" != "mainnet" ]; then
+  if ! grep -qE '^[[:space:]]*passphrase=' "$ENV_FILE"; then
+    echo "Error: 'passphrase' is not defined in $ENV_FILE"
+    exit 1
+  fi
+  if [ -z "${passphrase:-}" ]; then
+    echo "Error: 'passphrase' is empty in $ENV_FILE"
+    exit 1
+  fi
+fi
+
+dcld config broadcast-mode sync #TODO
+
+echo "Configure CLI"
+dcld config output json
+dcld config chain-id $chain_id
+dcld config node $node_endpoint
+
+random_four_digit_int() {
+  echo $(( RANDOM % 9000 + 1000 ))
+}
+
+# UPGRADE
+
+echo "Verify that upgrade is applied"
+result=$(dcld query upgrade applied $plan_name)
+echo "$result"
+
+test_divider
+
+########################################################################################
+
+echo "Verify that old data is not corrupted"
+
+# VENDORINFO
+
+echo "Verify if VendorInfo Record for VID: $vid is present or not"
+result=$(dcld query vendorinfo vendor --vid=$vid)
+check_response "$result" "\"vendorID\": $vid"
+check_response "$result" "\"companyLegalName\": \"$company_legal_name\""
+check_response "$result" "\"vendorName\": \"$vendor_name\""
+
+echo "Request all vendor infos"
+result=$(dcld query vendorinfo all-vendors)
+check_response "$result" "\"vendorID\": $vid"
+check_response "$result" "\"vendorID\": $vid_1"
+
+test_divider
+
+# MODEL
+
+echo "Get Model with VID: $vid PID: $pid_for_vid"
+result=$(dcld query model get-model --vid=$vid --pid=$pid_for_vid)
```

Comment:
```suggestion
result=$(dcld query model get-model --vid="$vid" --pid="$pid_for_vid")
```
It's better to use quoits for injection and mistake protection